internal/cli/arguments/arguments.go

```diff
@@ -34,18 +32,19 @@ func CheckFlagsConflicts(command *cobra.Command, flagNames ...string) {
 			return
 		}
 	}
-	feedback.Errorf(tr("Can't use %s flags at the same time.", "--"+strings.Join(flagNames, " "+tr("and")+" --")))
-	os.Exit(errorcodes.ErrBadArgument)
+	flags := "--" + strings.Join(flagNames, ", --")
```

Comment:
The translator will face the string `, %s` I don't know how many translators are able to get the logic behind it. Even if they got it right for some RTL languages may be tempting to translate it as `%s ,` but this will break the formatting.

Also, reading your link, looks like a comma is not only used for item lists.

So in this case I prefer to leave it as is, the string join will produce the list `--flag1, --flag2, --flag3`.